inc/namespace.php

```diff
@@ -50,3 +56,109 @@ function load_publication_checklist() {
+}
+
+/**
+ * Override the Duplicate Post supported post types if the config option defines new or different post types than the defautls.
+ *
+ * Default supported post types are 'post' and 'page'.
+ *
+ * Note: The configured option will _replace_ any enabled post types by default.
+ *
+ * @param array $enabled_post_types The post types that are supported by Duplicate Post.
+ * @return array The filtered array of supported post types.
+ */
+function maybe_override_enabled_post_types( array $enabled_post_types ) : array {
+	$post_types = Altis\get_config()['modules']['workflow']['clone-republish']['post-types'] ?? null;
+
+	if ( ! $post_types ) {
+		return $enabled_post_types;
+	}
+
+	return $post_types;
+}
+
+/**
+ * Remove the duplicate_post_original_item column.
+ *
+ * This column, on the post list, causes display/formatting issues and doesn't add any value, since the information is repeated in the title column.
+ *
+ * @param array $columns The array of post columns.
+ *
+ * @return array The filtered array of columns.
+ */
+function remove_duplicate_post_original_item_column( array $columns ) : array {
+	unset( $columns['duplicate_post_original_item'] );
+	return $columns;
+}
+
+/**
+ * Removes the "Original Post" column in the post list for all supported post types.
+ */
+function filter_duplicate_post_columns() {
+	$enabled_post_types = get_duplicate_post_types();
+	foreach ( $enabled_post_types as $post_type ) {
+		add_filter( "manage_{$post_type}_posts_columns", __NAMESPACE__ . '\\remove_duplicate_post_original_item_column', 11 );
+	}
+}
+
+/**
+ * Allow the roles defined in the Altis config to duplicate posts by default.
+ *
+ * @param mixed $roles The duplicate_post_roles value.
+ *
+ * @return array The filtered array of allowed roles.
+ */
+function filter_duplicate_post_roles( $roles ) : array {
+	$roles = Altis\get_config()['modules']['workflow']['clone-republish']['roles'] ?? $roles;
+
+	return $roles;
+}
+
+/**
+ * Exclude taxonomies defined in the Altis config, if specified.
+ *
+ * @param mixed $taxonomies An array of taxonomies to exclude from duplicated posts, if it exists.
+ *
+ * @return array The filtered array of excluded taxonomies.
+ */
+function filter_duplicate_post_excluded_taxonomies( $taxonomies ) : array {
+	$taxonomies = Altis\get_config()['modules']['workflow']['clone-republish']['taxonomies'] ?? [];
+
+	return $taxonomies;
+}
+
+/**
+ * Return an array of post types supported by Duplicate Post.
+ *
+ * @uses get_post_types() The get_post_types function must be loaded for this function to work properly.
+ *
+ * @return array The array of enabled post types.
+ */
+function get_duplicate_post_types() : array {
+	$public_post_types = get_post_types( [ 'public' => true ], 'names' );
+	$post_types = Altis\get_config()['modules']['workflow']['clone-republish']['post-types'] ?? $public_post_types;
+
+	return apply_filters( 'duplicate_post_enabled_post_types', $post_types );
```

Comment:
Been thinking about this - if we're applying the filter below then we don't need to check the config here, we can just pass all public post types as the default value e.g.
```suggestion

	return apply_filters( 'duplicate_post_enabled_post_types', $public_post_types );
```

Reply:
Yep. 👍 This is also somewhat duplicative of the function above that registers the post types from the config as enabled. This should really just be a helper function to get the enabled post types.